File: procedural-programming/homework-001/python/exercise_036.py
# Student: Pedro Henrique Resende Ribeiro
# Description: Print the volume of a cylinder
# Date: 2024-01-09 - Time: 01:30

PI = 3.141592

radius = float(input('Enter the radius of the cylinder: '))
height = float(input('Enter the height of the cylinder: '))

volume = PI * (radius ** 2.0) * height

print(f'The volume is {volume}')

File: procedural-programming/homework-001/c/exercise-036.c
/* Student: Pedro Henrique Resende Ribeiro
 * Description: Print the volume of a cylinder
 * Date: 2024-01-09 - Time: 01:30
 */

#include <math.h>
#include <stdio.h>

#define PI 3.141592

int main() {

    float radius, height, volume;

    printf("Enter the radius of the cylinder: ");
    scanf("%f", &radius);

    printf("Enter the height of the cylinder: ");
    scanf("%f", &height);

    volume = PI * pow(radius, 2.0) * height;

    printf("The volume is %f\n", volume);

    return 0;

}
